python/py-discogs-client/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-discogs-client
+python.rootname     python3-discogs-client
+version             2.3.12
+revision            0
+
+categories-append   devel audio
+license             BSD
+platforms           darwin
+supported_archs     noarch
+
+description         Continuation of the \"\"Official Python Client for the Discogs API\"
+long_description    This is the continuation of the official \"Discogs API client for Python\", \
+                    which was deprecated by discogs.com as of June 2020.
+
+homepage            https://github.com/joalla/discogs_client
+
+maintainers         {@catap korins.ky:kirill} openmaintainer
+
+checksums           rmd160  a06935ba7b98d1f49b86fa426a336738640a6819 \
+                    sha256  89e35ee23c43401d065d118d4be8d8b76214c8cc6c3e78bd4b9b45992258b1fe \
+                    size    32604
+
+python.versions     37 38 39
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_lib-append \
+                    port:py${python.version}-nose \
+                    port:py${python.version}-yaml \
+                    port:py${python.version}-docopt \
+                    port:py${python.version}-requests \
+                    port:py${python.version}-oauthlib \
+                    port:py${python.version}-sh \
+                    port:py${python.version}-dateutil
```

Comment:
I doubt that all these packages are actually need to install/run; some of them seem to be test dependencies as well? Upstreams [`setup.py`](https://github.com/joalla/discogs_client/blob/master/setup.py) doesn't list all of these - please verify.

Reply:
Well, I've used https://github.com/joalla/discogs_client/blob/master/requirements.txt as source. Which seems like dependencies which is required for tests: https://github.com/joalla/discogs_client/blob/master/.github/workflows/discogs_client-build.yml

=> very good catch. Fixed.